src/main/java/com/okta/developer/security/oauth2/ScimUserService.java

```diff
@@ -200,6 +200,7 @@ public FilterResponse<ScimUser> find(Filter filter, PageRequest pageRequest, Sor
             .map(user -> {
                 ScimUser scimUser = new ScimUser();
                 scimUser.setUserName(user.getLogin());
+                scimUser.setExternalId(user.getLogin());
```

Comment:
The calling client needs the `externalId` (i.e. it's userId, which matches the OIDC `sub` claim)